File: src/components/pages/projects/$projectId/shared/connect-snippets/apple/AppwriteClient.swift
import Appwrite
import Foundation

private func config(_ key: String) -> String {
  guard let value = Bundle.main.object(forInfoDictionaryKey: key) as? String
  else {
    fatalError("Missing \(key) in Info.plist")
  }
  return value
}

let client = Client()
  .setEndpoint(config("APPWRITE_ENDPOINT"))
  .setProject(config("APPWRITE_PROJECT_ID"))

File: src/components/pages/projects/$projectId/shared/connect-snippets/rust/actix/src/appwrite_client.rs
use appwrite::client::Client;

pub fn client() -> Client {
    Client::new()
        .set_endpoint(std::env::var("APPWRITE_ENDPOINT").unwrap())
        .set_project(std::env::var("APPWRITE_PROJECT_ID").unwrap())
        .set_key(std::env::var("APPWRITE_API_KEY").unwrap())
}
